stories/otp-input/otp-input.stories.mdx

```diff
@@ -62,13 +62,16 @@ you with it.
     <Story name="With custom action button">
         {() => {
             const [isInCooldown, setIsInCooldown] = useState(true);
+            const [otpValues, setOtpValues] = useState(new Array(4).fill(""));
```

Comment:
for a good example, set a const of `numOfInput = 4` then use it to form up the state and pass it to the component